fmriprep/cli/parser.py

```diff
@@ -5,7 +5,7 @@
 from .. import config
 
 
-def _build_parser():
+def _build_parser(**kwargs):
```

Comment:
Maybe add a docstring line stating `kwargs` are intended for `argparse.ArgumentParser`?